src/values.rs

```diff
@@ -67,11 +76,50 @@ impl Type {
         use self::Type::*;
         match *self {
             Duration | TimeStamp | Temperature | ExtNumeric | Color => false,
-            Unit | Bool | String | Json | Binary => true,
+            Unit | String | Json | Binary | OnOff | OpenClosed | ExtBool  => true,
         }
     }
 }
 
+/// An on/off state. Internal representation may be either On or Off.
+#[derive(Debug, Clone, Eq, PartialEq, Serialize, Deserialize)]
+pub enum OnOff {
+    On,
+    Off,
+}
+
+impl OnOff {
```

Comment:
Aaand few more ```best practices``` questions for you (sorry for too many, just want to have common ground) :)

* If I delegate ```PartialOrd``` and ```Ord``` implementation to ```#derive``` it won't behave like boolean (```true > false```, but ```OnOff::On < OnOff::Off```), I suspect we want to have boolean-like behavior and should implement ```Ord``` and ```PartialOrd``` manually?

* Is it ok to implement ```PartialOrd``` via ```Ord``` like:
```
impl PartialOrd for SomeEnum {
    fn partial_cmp(&self, other: &Self) -> Option<Ordering> {
        Some(self.cmp(other))
    }
}
```

Looking at the ```Ord``` and ```PartiallyOrd``` hierarchy it seems that more naturally would be to implement it the other way around, ```Ord``` via ```PartialOrd```, but we'll have addition ```unwrap``` call in ```self.partal_cmp(other).unwrap()```.

* Is it  Ok to use ```catch-all``` clause in two-states enum like this, or we should guard ourselves in case one more state will be added?
```
match (self, other) {
    (&OpenClosed::Open, &OpenClosed::Closed) => Greater,
    (&OpenClosed::Closed, &OpenClosed::Open) => Less,
    _ => Equal,
}
```
vs
```
match (self, other) {
    (&OpenClosed::Open, &OpenClosed::Closed) => Greater,
    (&OpenClosed::Closed, &OpenClosed::Open) => Less,
    (&OpenClosed::Open, &OpenClosed::Open) => Equal,
    (&OpenClosed::Closed, &OpenClosed::Closed) => Equal,
}
```
or even something crazy :)
```
match (self, other, self == other) {
    (_, _, true) => Equal,
    (&OpenClosed::Open, &OpenClosed::Closed, _) => Greater,
    (&OpenClosed::Closed, &OpenClosed::Open, _) => Less,
}
```